### PR TITLE
Display ecosystem information in spy mode

### PR DESCRIPTION
**Pull request by nathanstitt**
Todo:
 *  [x]  ireading step
 * [x] in the reference book

For assessments:
 * [x] ireadings
 * [x] homework
 * [x] practice

Uses BE PR: openstax/tutor-server#673

Reference View:
screen shot 2015-09-24 at 1 25 33 pm

Homework (Practice Widget is same):
screen shot 2015-09-24 at 1 58 23 pm

Reading:
screen shot 2015-09-24 at 1 59 01 pm

### CODE REVIEW

**Comment by Fredasaurus** [comment]
Who will be using this?

**Comment by nathanstitt** [comment]
It's for QA, and is only visible when the "spy mode" symbol is clicked

File: src/components/task/index.cjsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = React.createClass
   propTypes:
     id: React.PropTypes.string
 
-  displayName: 'ReadingTask'
+  displayName: 'Task'
```
Comment (nathanstitt) [inline src/components/task/index.cjsx]: The component was incorrectly named here.   This is the main component that wraps the steps.

**Comment by pandafulmanda** [comment]
c:sunglasses::sunglasses:l!

**Pull request merged**